.github/workflows/ci.yml

```diff
@@ -0,0 +1,191 @@
+      - name: "Build local docker image"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: mvn -B -U -q clean install -DadditionalOption=-Xdoclint:none -DskipTests -Dmaven.javadoc.skip=true -Dmaven.wagon.http.pool=false -Pbase
+      - name: "Cache LibreOffice"
+        run: bash _ci/cache_artifacts.sh
+      - name: "Run tests"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: mvn -B -U clean deploy -DadditionalOption=-Xdoclint:none -Dmaven.javadoc.skip=true -Dmaven.wagon.http.pool=false -Pbase
+
+  all_tests_matrix:
+    name: ${{ matrix.testName }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - testName: ImageMagick
+            buildProfile: imagemagick
+            testProfile: imagemagick
+          - testName: LibreOffice
+            buildProfile: libreoffice
+            testProfile: libreoffice
+          - testName: Transform Misc
+            buildProfile: misc
+            testProfile: misc
+          - testName: PDF Renderer
+            buildProfile: pdf-renderer
+            testProfile: pdf-renderer
+          - testName: Tika
+            buildProfile: tika
+            testProfile: tika
+          - testName: All in One Transformer
+            buildProfile: full-build
+            testProfile: aio-test
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.33.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.33.0
+      - name: "Login to Docker Hub"
+        uses: docker/login-action@v2.1.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: "Login to Quay.io"
+        uses: docker/login-action@v2.1.0
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: "Enable experimental docker features"
+        run: |
+          echo '{"experimental":true}' | sudo tee /etc/docker/daemon.json
+          sudo service docker restart
+      - name: "Build local docker image"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: bash _ci/build.sh ${{ matrix.buildProfile }}
+      - name: "Cache LibreOffice"
+        run: bash _ci/cache_artifacts.sh
+      - name: "Run tests"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: bash _ci/test.sh ${{ matrix.testProfile }}
+
+  release:
+    name: "Release"
+    runs-on: ubuntu-latest
+    needs: [veracode, build_and_test, all_tests_matrix]
+    if: >
+      !failure() &&
+      contains(github.event.head_commit.message, '[release]') &&
+      github.event_name != 'pull_request' &&
+      (github.ref_name == 'master' || contains(github.ref_name, 'SP/') || contains(github.ref_name, 'HF/'))
```

Comment:
In several places in this PR we're using this syntax, but I believe we really mean `startsWith` instead of `contains`?